inference-engine/tools/compile_tool/README.md

```diff
@@ -1,6 +1,11 @@
 # Compile Tool {#openvino_inference_engine_tools_compile_tool_README}
 
-The Compile tool is a C++ application that enables you to dump a loaded executable network blob.
+The Compile tool is a C++ application that enables you to compile a network for inference on Intel® Neural Compute Stick 2 and export it to a binary file. With the Compile Tool, you can compile a network on a machine that doesn't have the physical device connected and then transfer a generated file to any machine with the target Intel® Neural Compute Stick 2 device available.
```

Comment:
It is not limited to `Intel® Neural Compute Stick 2`. It can work with other plugins, which supports import/export (like FPGA). I would avoid using actual device names here and create separate paragraph with supported devices list.

Reply:
So, what are the supported plugins? Documentation is version-specific, so we shouldn't mention FPGA in this list, as it's no longer available in OpenVINO releases since 2020.4 (but available in LTS releases only - see the Note added below).

Reply:
OK, so for now this list will contain one item (MYRIAD plugin). But we can extend it in future and do it in single place.